# EBI workshop: Extracting information from omics data
Part of the [Data-driven approaches to understanding dementia](https://www.ebi.ac.uk/training/events/data-driven-approaches-understanding-dementia/#vf-tabs__section--tab2) course. The session titled "Extracting information from omics data"
# Lecture

# Practical workshop:
We are focusing on the data from:
Yu, Min, et al. "Integrative multi-omic profiling of adult mouse brain endothelial cells and potential implications in Alzheimerâ€™s disease." Cell reports 42.11 (2023).

In this workshop we will explore some of this data together


### 0. Let's look at pre-processing of the RNA-Seq and ATAC-Seq from this paper
[Slides to follow along.](https://docs.google.com/presentation/d/1MSrh01gnCiXreGWjnIFtRNezeCu32LW2NSv5sX_ZcOc/edit?usp=sharing)

Next we will get started with some downstream analysis.
### 1. Fork this repo!
/Quick-start guide to how Git works/

Fork the dev branch of this repo to your Github account. This gives you your own copy to play with that doesn't have any impact on this main repository. As we proceed with the analysis we encourage you to make your own notes and comments in the document and commit them as you go.
### 2. Try to run the first code block..
Ah.. is there any problem? Correct the error and make your first commit!
### 3. Let's proceed with some analysis to identify the brain-specific endothelial genes
Workshop.qmd
### 4. Now we have a gene list let's see if any of these genes are enriched in disease GWAS
blahblah.qmd
### 5. Let's see if groups of genes might work together in regulatory units

### 6. Is our narrowed down list more biologically meaningful?
### 7. Make a pull request to this repo!
This whole time you have been commiting notes and comments to your forked version of our code. Now try making a pull request to this repository, we will review the request and either approve or deny it.

